ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -79,6 +94,108 @@
         }
     }
 
-    Utils.initializeEditDialog(EDIT_DIALOG)(handleAsyncSubmissionAndThankYouOption, handleSubmitAction);
+    function loadSubmitActionSubDialogClientLibs(dialog) {
+        var containerSubmitAction = dialog.find(CONTAINER_SUBMITACTION)[0];
+        containerSubmitAction.on("change", function(){
+            loadRestEndPointDialogClientlibs(dialog);
+            loadFDMDialogClientlibs(dialog);
+            loadEmailDialogClientlibs(dialog);
+        });
+    }
+
+    function showDorBindRefSelector (dialog) {
+        var enableDORSubmissionCheckBoxElement = dialog.find(FDM_ENABLE_DOR_CHECK_BOX)[0],
+            $dorBindRefSelector = Utils.selectElement("input", './bindRef'),
+            parentBindRefSelector = "span.bindRefSelector";
```

Comment:
these are internal selectors. we should be clear about the internal and public classification. if we are implementing a dialog inside the product (outside of core component module). then all logic for interaction must be internal.

Reply:
I tried to reuse the functions we have defined internally, if these are not meant to be exposed, I can write custom functions for these.

Reply:
removed